PasswordRulesSharp/Builder/RuleBuilder.cs

```diff
@@ -49,32 +75,74 @@ public RuleBuilder Require(CharacterClass characterClass)
         return this;
     }
 
+    /// <summary>
+    /// Appends a <see cref="Rule.Required"/> requirement for <c>Unicode</c> to
+    /// the rule you're building. At least one character must be a Unicode
+    /// character.
```

Comment:
Maybe also specify what unicode characeters include. IIRC this doesn't exclude any characters and implicitly means "any character".